db/seeds/priority_distributions.rb

```diff
@@ -0,0 +1,384 @@
+    end
+
+    def create_ama_hearing_genpop_cases
+      create_ama_hearing_ready_priority_genpop_cases
+      create_ama_hearing_ready_nonpriority_genpop_cases
+      create_ama_hearing_nonready_priority_genpop_cases
+    end
+
+    def create_direct_review_genpop_cases
+      create_direct_review_ready_priority_genpop_cases
+      create_direct_review_ready_nonpriority_genpop_cases
+      create_direct_review_nonready_priority_genpop_cases
+    end
+
+    def create_evidence_submission_genpop_cases
+      create_evidence_submission_ready_priority_genpop_cases
+      create_evidence_submission_ready_nonpriority_genpop_cases
+      create_evidence_submission_nonready_priority_genpop_cases
+    end
+
+    def create_priority_distribution_this_month(judge)
+      create_unvalidated_completed_distribution(
+        traits: [:priority, :this_month],
+        judge: judge,
+        statistics: { "batch_size" => rand(10) }
+      )
+    end
+
+    def create_priority_distribution_last_month(judge)
+      create_unvalidated_completed_distribution(
+        traits: [:priority, :last_month],
+        judge: judge,
+        statistics: { "batch_size" => rand(10) }
+      )
+    end
+
+    def create_nonpriority_distribution_this_month(judge)
+      create_unvalidated_completed_distribution(
+        traits: [:this_month],
+        judge: judge,
+        statistics: { "batch_size" => rand(10) }
+      )
+    end
+
+    def create_legacy_ready_priority_cases_tied_to_judge(judge)
+      rand(5).times do
+        create(
+          :case,
+          :aod,
+          :ready_for_distribution,
+          :tied_to_judge,
+          tied_judge: judge,
+          bfkey: random_key,
+          correspondent: create(:correspondent, stafkey: random_key)
+        )
+      end
+    end
+
+    def create_legacy_nonready_priority_cases_tied_to_judge(judge)
+      rand(5).times do
+        create(
+          :case,
+          :aod,
+          :tied_to_judge,
+          tied_judge: judge,
+          bfkey: random_key,
+          correspondent: create(:correspondent, stafkey: random_key)
+        )
+      end
+    end
+
+    def create_legacy_ready_nonpriority_cases_tied_to_judge(judge)
+      rand(5).times do
+        create(
+          :case,
+          :ready_for_distribution,
+          :tied_to_judge,
+          tied_judge: judge,
+          bfkey: random_key,
+          correspondent: create(:correspondent, stafkey: random_key)
+        )
+      end
+    end
+
+    def create_hearing_ready_priority_cases_tied_to_judge(judge)
+      rand(5).times do
+        create(
+          :appeal,
+          :hearing_docket,
+          :ready_for_distribution,
+          :advanced_on_docket_due_to_age,
+          :held_hearing,
+          :tied_to_judge,
+          tied_judge: judge,
+          adding_user: User.first
+        )
+      end
+    end
+
+    def create_hearing_nonready_priority_cases_tied_to_judge(judge)
+      rand(5).times do
+        create(
+          :appeal,
```

Comment:
Suggestion: More realistic not-ready appeal:
```suggestion
          :appeal,
          :with_post_intake_tasks,
```

Reply:
same for the other `nonready` `ama` situations